core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -590,6 +509,58 @@ class TeamPolicyInternal<Kokkos::Cuda, Properties...>
     return *this;
   }
 #endif
+
+  template <class FunctorType, class ClosureType>
+  int internal_team_size_max<closure_type>(const FunctorType& f) {
```

Comment:
remove the `<closure_type>` here and add a `using closure_type=ClosureType` later